src/index.html

```diff
@@ -11,7 +11,219 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-div-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset class="personalinfo">
+        <legend>
+          Personal information
+        </legend>
+
+        <label>
+          Surname:  
+          <input 
+            type="text"
+            class="surname" 
+            name="surname" 
+            required
+          >
+        </label>
+
+        <div class="one">  
+          <label>
+            Name:  
+            <input 
+              type="text"
+              class="name"
+              name="name" 
+              required
+            >      
+          </label>
+        </div>
+        
+        <div class="one">
+          <label>
+            How old are You?
+            <input 
+              type="number"
+              class="howold" 
```

Comment:
Do you really need different classes for each input? You don't use them, as I see. Fix it everywhere
```suggestion
```